ground/uavobjgenerator/main.cpp

```diff
@@ -29,7 +29,7 @@
 #include <QStringList>
 #include <iostream>
 
-#include <unistd.h>	/* For symlink(2) */
+//#include <unistd.h>	/* For symlink(2) */
```

Comment:
@mlyle added it for reasons I can't explain, I'll remove it after he reviews this.